src/AddTaskPage.js

```diff
@@ -6,8 +6,16 @@ export function AddTaskPage(props) {
         <div>
             <label id="addTask" htmlFor="newTask">Add New Task:</label>
             <input type="text" id="newTask" name="newTask"/>
+            <select name="priority" id="priority-dropdown">
+                <option value="1">High</option>
+                <option value="2">Medium</option>
+                <option value="3">Low</option>
+            </select>
+            <input type="button" value="Cancel"
+                   onClick={() => props.cancel()}/>
             <input type="button" value="Add"
-                   onClick={() => props.addNewDataPoint(document.getElementById("newTask").value)}/>
+                   onClick={() => props.addNewDataPoint(document.getElementById("newTask").value,
```

Comment:
Do not use document.getElementById. Instead
* have state information containing: the text of the task, the priority
* have the text and menu elements onChange methods updated the state.
*  and then have the onClick retrieve those state values.